src/main/java/net/skinsrestorer/shared/utils/PlayerWrapper.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * #%L
+ * SkinsRestorer
+ * %%
+ * Copyright (C) 2021 SkinsRestorer
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+package net.skinsrestorer.shared.utils;
+
+/**
+ * Makes it possible to get all platforms into a single API merged.
+ */
+public class PlayerWrapper {
```

Comment:
I don't know what a PlayerWrapper is.
Does it support illegal usernames?

Reply:
I guess any player instance possible. Its just like a storage for any type of player on any platform.